agent/php_nrini.c

```diff
@@ -1730,6 +1730,45 @@ static PHP_INI_MH(nr_unsigned_int_mh) {
   return SUCCESS;
 }
 
+static PHP_INI_MH(nr_log_events_max_samples_stored_mh) {
+  nriniuint_t* p;
+  int val = NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED;
+
+#ifndef ZTS
+  char* base = (char*)mh_arg2;
+#else
+  char* base = (char*)ts_resource(*((int*)mh_arg2));
+#endif
+
+  p = (nriniuint_t*)(base + (size_t)mh_arg1);
+
+  (void)entry;
+  (void)mh_arg3;
+  NR_UNUSED_TSRMLS;
+
+  /* Anything other than a valid value will result in the
+   * default value of NR_DEFAULT_LOG_EVENTS_MAX_SAMPLES_STORED.
```

Comment:
Would be good to come up with how code will handle:
- value < 0
- value >  maximum allowed log max sampled stored (prob need to figure out this value as well)
-

Reply:
See https://github.com/newrelic/newrelic-php-agent/pull/488#discussion_r925868034